tests/TestCase/Middleware/DebugKitMiddlewareTest.php

```diff
@@ -125,7 +125,7 @@ public function testInvokeSaveData()
         $this->assertNotNull($result->panels[11]->summary);
         $this->assertSame('Sql Log', $result->panels[11]->title);
 
-        $timeStamp = filemtime(Plugin::path('DebugKit') . 'webroot' . DS . 'js' . DS . 'main.js');
+        $timeStamp = filectime(Plugin::path('DebugKit') . 'webroot' . DS . 'js' . DS . 'main.js');
```

Comment:
It should be `inject-iframe.js` here and keep using `filemtime`.

Line was previously incorrectly updated when .js file was renamed. And `ToolbarService::getToolbarUrl‎` is using `filemtime`.

Reply:
I tried that but for some reason its not working... haven't looked too deep into it though.